doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_arcgis_featuretable_sink.md

```diff
@@ -40,29 +42,75 @@ Argis feature tables must be provisioned before sending entities.
 
-Unique field is provided to allow `NGSIArcgisFeatureTableSink` to update existant entities. NGSI `entity type` will be used as unique field name.
+Unique field is provided to allow `NGSIArcgisFeatureTableSink` to update existant entities. NGSI `entity type` will be used as unique field name. This means that a feature named `type` in the Feature Table cannot be filled in by the sink. If Feature Table needs to persist the value of entity type it has to be in a feature different than `type`.
 
-All this parameters, can be customized using Cygnus mapping capabilities.
+All this parameters, can be customized using Cygnus mapping capabilities or Context Broker custom notifications.
+
+##### Using Cygnus
 
 Let's see an example:	
 
-##### Agent.conf file:
+###### Agent.conf file:
 
 	agent.arcgis-sink.arcgis_service_url = https://arcgis.com/{hash}/arcgis/rest/services
-##### Entity data:
+###### Entity data:
 
 	service = vehicles
 	service-path = /4wheels
 	entity-type = car
-##### result
+###### result
 
 	Feature table url: https://arcgis.com/{hash}/arcgis/rest/services/vehicles/4wheels
 	Table's unique field: car
 
 [Top](#top)
 
+#### Using Context Broker custom notifications
+
+If a service path has different entitiy types, following the previous example `car` and `van`, they will need to persist in different Feature Layers, to avoid the limitation of one "fiware-servicepath" can only persist in a unique feature table, it can be used [CB custom notifications](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#custom-notifications) in which we can map a different value for `fiware-service` and `fiware-servicepath` headers parameters. 
+
+###### Entity data:
+
+	service = vehicles
+	service-path = /4wheels
+	entity-type = van
+
+For example, if the Feature table for "van" is `https://arcgis.com/{hash}/arcgis/rest/services/vehicles/van`,  the custom notif could be:
+```
+"httpCustom":{
+  "url": "http://iot-cygnus:<source_arcgis_port>/notify",
+  "headers": {
+    "fiware-service": "vehicles",
+    "fiware-servicepath": "/van"
+  }
+}
+```
+
+Using this functionality a different "fiware-servicepath" can be assigned to every entity type.
+
+Moveover, to modify the unique field value of `type` attribute it can be used the [ngsi patching functionality](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#ngsi-payload-patching):
+
+```
+"httpCustom":{
+  "url": "http://iot-cygnus:<source_arcgis_port>/notify",
+  "ngsi": {
+    "type": "car"
+  },
+  "headers": {
+    "fiware-service": "vehicles",
+    "fiware-servicepath": "/van"
+  }
+}
+```
+
+###### result
+
+	Feature table url: https://arcgis.com/{hash}/arcgis/rest/services/vehicles/van
+	Table's unique field: car
```

Comment:
This is the result of the second example? Why don't include the result of the first example?

It's a bit weird... maybe we should talk by skype.